Description: Fix the program names that appear in the manpages.
 Manpages are generated by help2man, which calls the program with --help.
 Because of libtool, the program names get lt- prepended which is both 
 wrong and ugly in the manpages. Remove that prefix.
Author: Nicolas Boullis <nboullis@debian.org>
--- a/src/cd-drive.1
+++ b/src/cd-drive.1
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.38.4.
-.TH LT-CD-DRIVE "1" "October 2011" "lt-cd-drive version 0.83 i686-pc-linux-gnu" "User Commands"
+.TH CD-DRIVE "1" "October 2011" "cd-drive version 0.83 i686-pc-linux-gnu" "User Commands"
 .SH NAME
-lt-cd-drive \- manual page for lt-cd-drive version 0.83 i686-pc-linux-gnu
+cd-drive \- manual page for cd-drive version 0.83 i686-pc-linux-gnu
 .SH SYNOPSIS
 .B cd-drive
 \fIOPTION\fR...
--- a/src/cd-info.1
+++ b/src/cd-info.1
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.38.4.
-.TH LT-CD-INFO "1" "October 2011" "lt-cd-info version 0.83 i686-pc-linux-gnu" "User Commands"
+.TH CD-INFO "1" "October 2011" "cd-info version 0.83 i686-pc-linux-gnu" "User Commands"
 .SH NAME
-lt-cd-info \- manual page for lt-cd-info version 0.83 i686-pc-linux-gnu
+cd-info \- manual page for cd-info version 0.83 i686-pc-linux-gnu
 .SH SYNOPSIS
 .B cd-info
 \fIOPTION\fR...
--- a/src/cd-read.1
+++ b/src/cd-read.1
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.38.4.
-.TH LT-CD-READ "1" "October 2011" "lt-cd-read version 0.83" "User Commands"
+.TH CD-READ "1" "October 2011" "cd-read version 0.83" "User Commands"
 .SH NAME
-lt-cd-read \- manual page for lt-cd-read version 0.83
+cd-read \- manual page for cd-read version 0.83
 .SH SYNOPSIS
 .B cd-read
 \fIOPTION\fR...
--- a/src/iso-info.1
+++ b/src/iso-info.1
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.38.4.
-.TH LT-ISO-INFO "1" "October 2011" "lt-iso-info version 0.83 i686-pc-linux-gnu" "User Commands"
+.TH ISO-INFO "1" "October 2011" "iso-info version 0.83 i686-pc-linux-gnu" "User Commands"
 .SH NAME
-lt-iso-info \- manual page for lt-iso-info version 0.83 i686-pc-linux-gnu
+iso-info \- manual page for iso-info version 0.83 i686-pc-linux-gnu
 .SH SYNOPSIS
 .B iso-info
 \fIOPTION\fR...
--- a/src/iso-read.1
+++ b/src/iso-read.1
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.38.4.
-.TH LT-ISO-READ "1" "October 2011" "lt-iso-read version 0.83 i686-pc-linux-gnu" "User Commands"
+.TH ISO-READ "1" "October 2011" "iso-read version 0.83 i686-pc-linux-gnu" "User Commands"
 .SH NAME
-lt-iso-read \- manual page for lt-iso-read version 0.83 i686-pc-linux-gnu
+iso-read \- manual page for iso-read version 0.83 i686-pc-linux-gnu
 .SH SYNOPSIS
 .B iso-read
 \fIOPTION\fR...
